src/Aspire.Hosting.Redis/RedisBuilderExtensions.cs

```diff
@@ -55,22 +55,34 @@ public static IResourceBuilder<RedisResource> WithRedisCommander(this IResourceB
     }
 
     /// <summary>
-    /// Adds a named volume for the data folder to a Redis container resource.
+    /// Adds a named volume for the data folder to a Redis container resource and enables Redis persistence.
     /// </summary>
     /// <param name="builder">The resource builder.</param>
     /// <param name="name">The name of the volume. Defaults to an auto-generated name based on the resource name. </param>
     /// <param name="isReadOnly">A flag that indicates if this is a read-only volume.</param>
     /// <returns>The <see cref="IResourceBuilder{T}"/>.</returns>
     public static IResourceBuilder<RedisResource> WithDataVolume(this IResourceBuilder<RedisResource> builder, string? name = null, bool isReadOnly = false)
-        => builder.WithVolume(name ?? $"{builder.Resource.Name}-data", "/data", isReadOnly);
+        => builder.WithVolume(name ?? $"{builder.Resource.Name}-data", "/data", isReadOnly)
+                  .WithPersistence(TimeSpan.FromSeconds(60));
```

Comment:
Why a default value?

Reply:
I think it's reasonable to have a default here but to your point it should likely be defined by the WithPersistence method and just documented here. Folks can call WithPersistence with args to set custom values.

Reply:
I moved the defaults to `WithPersistence` and updated the doc comments to make it clearer how to change the persistence configuration.